src/content/getting-started/exposing-workloads/index.md

```diff
@@ -117,8 +123,6 @@ Once the ingress is up, you will be able to access your service publicly at a UR
 http://PREFIX.CLUSTER_ID.k8s.gigantic.io
 ```
 
-Currently, this is limited to exposing by default on port `80`. Support for TLS will be added soon.
-
 For additional features and options, please see our documentation around [advanced ingress configuration]({{< relref "/advanced/ingress/configuration" >}}).
 
 ## Forwarding an authenticated port with `kubectl port-forward` {#port-forward}
```

Comment:
@giantswarm/team-ludacris since you're listed as doc owners

what do you think about removing the sections about port-forwarding and accessing the API directly? IMO this information is (a) slightly out of date (b) available somewhere else (c) a little missleading, since using Ingress should be the way to go for exposing stuff

WDYT?

Reply:
Yes IMO we should get rid of this / make it a lot shorter by just linking to upstream `kubectl port-forward` is anything. I would rather not have anyone use it.

Reply:
Thinking about it more:
Should these docs be complete ways of how you _can_ expose something or what customer _should_ use in testing / prod?

Reply:
Mh unsure what the intention of the original author was. IMO it should be the latter.

Reply:
OA here ;) intention back then was indeed "exposing sth" but that has changed over the years, I do think having this IC focused makes sense and the other methods are kinda anyway explained upstream and by now also less useful.